crates/typst-library/src/model/outline.rs

```diff
@@ -121,21 +207,22 @@ pub struct OutlineElem {
 
     /// How to indent the outline's entries.
     ///
-    /// - `{none}`: No indent
-    /// - `{auto}`: Indents the numbering of the nested entry with the title of
-    ///   its parent entry. This only has an effect if the entries are numbered
-    ///   (e.g., via [heading numbering]($heading.numbering)).
-    /// - [Relative length]($relative): Indents the item by this length
-    ///   multiplied by its nesting level. Specifying `{2em}`, for instance,
-    ///   would indent top-level headings (not nested) by `{0em}`, second level
+    /// - `{auto}`: Indents the numbering/prefix of a nested entry with the
+    ///   title of its parent entry. If the entries are not numbered (e.g., via
+    ///   [heading numbering]($heading.numbering)), this instead simply inserts
+    ///   a fixed amount of `{1.2em}` indent per level.
+    ///
+    /// - [Relative length]($relative): Indents the entry by the specified
+    ///   length per nesting level. Specifying `{2em}`, for instance, would
+    ///   indent top-level headings (not nested) by `{0em}`, second level
```

Comment:
```suggestion
    ///   indent top-level headings by `{0em}` (not nested), second level
```

This is more consistent with the rest of the sentence.